### DISCUSSION TITLE
Get original blog name regardless of reblog status

### DISCUSSION THREAD

**Discussion by federatedeagle** [TumblThreeApp/TumblThree#516] [General]
For the Filename Template, in order to get the blog title blog there are 2 options
%b - Blog name
%o -Blog name of reblog origin
I want to get the original blog name always, so use reblog origin name if it is a reblog , and if its an original post I would want the regular blog name. Some thing like
IF
Reblog = 1 then %o
ELSE %b

Including both %b and %o in the template right now results in reblogs having both the reblogger and the original blog name in the title.
Is something like this possible right now?
Thanks!

**Comment by thomas694** [comment]
This is currently not possible. But it could come in one of the next versions.

**Comment by thomas694** [comment]
In the new version there's a new token `%q`.
